tests/test_tti.py

```diff
@@ -113,15 +113,23 @@ def test_tti_staggered(shape):
     solver_tti2 = AnisotropicWaveSolver(model, geometry, time_order=2, space_order=8)
 
     # Solve
-    configuration['dse'] = 'aggressive'
-    configuration['dle'] = 'advanced'
-    rec1, u1, v1, _ = solver_tti.forward(kernel='staggered')
-    configuration['dle'] = 'basic'
-    rec2, u2, v2, _ = solver_tti2.forward(kernel='staggered')
+    @switchconfig(dse='aggressive', dle='advanced')
+    def run_tti(solver):
+        rec1, u1, v1, _ = solver.forward(kernel='staggered')
+        return rec1, u1, v1
+
+    rec1, u1, v1 = run_tti(solver_tti)
+
+    @switchconfig(dse='aggressive', dle='basic')
+    def run_tti2(solver):
```

Comment:
now that I think about it, if this is just a DSE/DLE test, why not moving it to test_dse or test_dle ? I guess I'm a bit confused

Reply:
Done